### ISSUE TITLE
[BUG]: TTA ldm training loss

### ISSUE THREAD

**Issue by Sreyan88** [open-mmlab/Amphion#162]
## Describe the bug
The loss for ldm (TTA) is calculated between the model output and pure noise [here](https://github.com/open-mmlab/Amphion/blob/main/models/tta/ldm/audioldm_trainer.py#L200). Is this right? I think the loss should be calculated with the ground truth latent.

## How To Reproduce
The same steps as mentioned in the repo for TTA.

## Expected behavior
Mentioned above.

## Screenshots
N/A

## Environment Information
The same steps as mentioned in the repo for TTA.

## Additional context
N/A
